js/src/admin/components/AdminPage.tsx

```diff
@@ -75,6 +76,7 @@ export interface HTMLInputSettingsComponentOptions extends CommonSettingsItemOpt
 
 const BooleanSettingTypes = ['bool', 'checkbox', 'switch', 'boolean'] as const;
 const SelectSettingTypes = ['select', 'dropdown', 'selectdropdown'] as const;
+const ColorSettingTypes = ['color-input', 'color-preview', 'custom-color'] as const;
```

Comment:
Do we need to have `color-input` in here? The other two are self explanatory, but this one is a bit confusing

Reply:
I'd prefer only having one value, to be honest. For 2.0 we should deprecate the others.

Reply:
If we make it just `color-preview`, I assume that should be changed to this?

```js
const ColorPreviewSettingType = 'color-preview' as const
```

Reply:
Makes sense to me, although all the current checks for characters will need to be changed.

Also, is `as const` still necessary then?

Reply:
Oh, looks like it is not. I had no idea what it did until now tbh.